spec/features/user_resets_password_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+feature "user resets password" do
+  scenario "user resets password" do
+    
+    user = Fabricate(:user)
+    
+    visit sign_in_path
+    click_reset_password
+    assert_reset_password_page
+
+    enter_and_submit_email(user)
+    assert_confirmation_page
```

Comment:
Did you mean to name this `assert_confirmation_message`? That would make more sense based on the actual assert in the method.